### DISCUSSION TITLE
Improving usability in the Open File dialog window

### DISCUSSION THREAD

**Discussion by benlazaro** [otros-systems/otroslogviewer#628] [Ideas]
I think we can improve the usability in the Open File dialog window by allowing the user to select the type of VFS that they wants to use.
For example, we know what are the types of file systems that Apache VFS supports, so we could should a drop-down field which enumerates the supported files systems. 
We could also display configuration options applicable to the selected FS, such as "Do not follow SFTP links", etc.
This makes the open file window more intuitive and easier to use.

**Comment by svennissel** [comment]
We can create drop down only to fill a example in the location input file like this: 
![grafik](https://user-images.githubusercontent.com/40182921/166138786-97ebf4ba-bcce-46ba-aa78-0785e3d85bcd.png)

Or more something like this:
![grafik](https://user-images.githubusercontent.com/40182921/166138909-2106c82f-a6ad-4aca-89fa-b39d56e6c88c.png)
